terraform/shared/shared.tf.json.template.py

```diff
@@ -49,6 +58,15 @@
                 }
             }
         },
+        'aws_s3_bucket_logging': {
+            'shared_cloudtrail': {
```

Comment:
I thought we agreed to make this conditional so that it would not be created for `dev.shared` where this would fail to be created.

Reply:
Yes of course. Sorry, lost my change when I tried to factor out the FIXMEs into their own commit.